net-misc/loki-promtail-bin/files/promtail.yml

```
@@ -0,0 +1,18 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
```

We don't need promtail running as a server. Let's disable it
```suggestion
 disable: true
```